lib/modules/versioning/ubuntu/index.spec.ts

```diff
@@ -195,9 +196,17 @@ describe('modules/versioning/ubuntu/index', () => {
     ${'groovy'}   | ${false}
     ${'hirsute'}  | ${false}
     ${'impish'}   | ${false}
-    ${'jammy'}    | ${false}
+    ${'jammy'}    | ${true}
   `('isStable("$version") === $expected', ({ version, expected }) => {
-    expect(ubuntu.isStable(version)).toBe(expected);
+    const orgNow = Settings.now;
+    const dt = DateTime.fromISO('2022-04-30');
```

Comment:
```suggestion
    const dt = DateTime.fromISO('2022-03-30');
```

Set a test date before latest release, so we can test the unreleased code path again

Reply:
Good idea